content/docs/reference-react.md

```diff
@@ -88,6 +92,42 @@ If your React component's `render()` function renders the same result given the
 
 * * *
 
+### `React.memo`
+
+```javascript
+const MyComponent = React.memo(function(props) {
+  /* render using props */
+});
+```
+
+`React.memo` is a [higher order component](/docs/higher-order-components.html). It's similar to [`React.PureComponent`](#reactpurecomponent) but for function components instead of classes.
+
+If your function component renders the same result given the same props, you can wrap it in a call to `React.memo` for a performance boost in some cases by memoizing the result.
```

Comment:
Our readers probably don't know what "memoize" means. Maybe add

(This means that React will skip rendering the component, and reuse the last rendered result.)